stackdriver_trace.c

```diff
@@ -30,7 +30,7 @@
 #include <sys/time.h>
 #endif
 
-// True global for storing the original zend_execute_ex function pointer
+/* True globals for storing the original zend_execute_ex and zend_execute_internal function pointers */
```

Comment:
Looks like this line is longer than 80 chars. What is the maximum line length used in the current style guide?